priv/plts/dialyzer.plt.hash

```diff
@@ -0,0 +1 @@
+ǒR)��T�?�>�Oh �
```

Comment:
Looks like the files `priv/plts/dialyzer*` were mistakenly checked in. Can try to remove using

```
$ git rm --cached priv/plts/dialyzer.plt*
```

Reply:
done